### DISCUSSION TITLE
Parameters for training ResNet-50 backbone

### DISCUSSION THREAD

**Discussion by dani3l125** [STVIR/pysot#552] [Q&A]
Hello,

I'm trying to fine-tune the backbone of SiamMask (on ImageNet-1k) after performing some operations on the network.
I would like to know where can I find the parameters of the original training, in particular: torch.transforms.Normalize and other transforms of the data, epochs number and learning rate...  

Thank you in advance,
Daniel
